kale/loaddata/few_shot.py

```diff
@@ -0,0 +1,124 @@
+"""
+Dataset for N-way-K-shot setting in few-shot problems.
```

Comment:
?

Reply:
Changed to _Dataset class to load data for few-shot learning problems under N-way-K-shot settings._